modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/internal/classloader/DefaultArtifactClassLoaderResolver.java

```diff
@@ -116,8 +114,7 @@ public MuleDeployableArtifactClassLoader createDomainClassLoader(DomainDescripto
 
     regionClassLoader.addClassLoader(domainClassLoader, artifactClassLoaderFilter);
 
-    List<ArtifactPluginDescriptor> artifactPluginDescriptors =
-        removeExportedPackagesAlreadyExportedByTransitiveDependencies(process(descriptor.getPlugins(), false, List::add));
+    List<ArtifactPluginDescriptor> artifactPluginDescriptors = process(descriptor.getPlugins(), false, List::add);
```

Comment:
Keeping this because although plugins have been already ordered, they are in a `Set`, so here we guarantee the ordered list needed for the class loaders creation.